### PR TITLE
how-to-review-a-pull-request

### LINKED ISSUE #3054: Document How to Review branches

### PR DESCRIPTION
**Pull request by shanshanhsieh**
this pr resolves #3054 

to test, read the documentation in the file changes, and comment on the parts that are missing or unclear.

### CODE REVIEW

File: docs/developer/how-to-review-a-pull-request.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+instruction is unclear.
+Once the test provided by the author has passed, the reviewer may proceed to the next step.
+
+2. Go through the file changes
+------------------------------
+
+It is always a good idea to go through all the changes at least once.
+Please follow this guide_ to review the file changes on GitHub.
+
+.. _guide: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request>
```
Comment (lguilhermers) [inline docs/developer/how-to-review-a-pull-request.rst:23]:
```suggestion
_guide: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request
```

File: docs/developer/how-to-review-a-pull-request.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+3. Run tests
+------------
+All PR is automatically sent to test by the Jenkins_, it executes ``cea test --workflow quick`` on a remote computer.
+The test results is directly shown in the PR page on GitHub.
+
+Additionally, it is always a good idea to run a complete test (``cea test --workflow slow``) on your local computer.
+If Jenkins encounter any errors, you can also reproduce those errors by running ``cea test --workflow quick`` locally.
+See here_ for more information on ``cea test``.
+
+.. _here: :doc:`how-to-test-the-cea`
```
Comment (lguilhermers) [inline docs/developer/how-to-review-a-pull-request.rst:46]: link not working

**Review by lguilhermers — CHANGES_REQUESTED** [review]
Looks good :)
Added a couple of comments, which was also good to test those features myself.

File: docs/tutorials.rst [hunk 3]
```diff
@@ -22,13 +22,11 @@ These are a collection of tutorials we consider essential for any new developer
 #. :doc:`how-to-add-a-new-script-to-the-cea`
 #. :doc:`how-to-test-the-cea`
 #. :doc:`how-to-set-up-nsis`
-#. :doc:`how-to-create-a-new-release`
```
Comment (MatNif) [inline docs/tutorials.rst:25]: I'm not familiar with these two files unfortunately. Why did you decide to remove them?
Are they obsolete?

Comment (shanshanhsieh) [reply docs/tutorials.rst:25]: those links are broken, and they are already linked somewhere else: `[docs](https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/master/docs)/[developer](https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/master/docs/developer)/developer-walk-through.rst`

Comment (MatNif) [reply docs/tutorials.rst:25]: ok, perfect. In that case, the PR can be merged in my opinion @lguilhermers

File: docs/developer/how-to-review-a-pull-request.rst [hunk 4]
```diff
@@ -20,30 +20,30 @@ Once the test provided by the author has passed, the reviewer may proceed to the
 It is always a good idea to go through all the changes at least once.
 Please follow this guide_ to review the file changes on GitHub.
 
-.. _guide: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request>
+_guide: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request
```
Comment (lguilhermers) [inline docs/developer/how-to-review-a-pull-request.rst:23]:
````
.. _guide:https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request
````

**Review by lguilhermers — COMMENTED** [review]
Looks good to me, apart from the "guide" hyperlink which needs to be fixed.

**Review by lguilhermers — APPROVED** [review]

**Comment by shanshanhsieh** [comment]
@lguilhermers , please merge the pr if everything is alright, thanks!

**Comment by lguilhermers** [comment]
> @lguilhermers , please merge the pr if everything is alright, thanks!

Ah, ok. I was thinking the common practice is for whoever published the PR to have the final call and merge.
Thanks for clarifying.

**Pull request merged**